### PR TITLE
AB#88304 fixed site integration tests

### PR DESCRIPTION
**Pull request by rafalgajewski**
…ges in test

### CODE REVIEW

File: src/HE.Investment.AHP.Domain/Site/CommandHandlers/SiteBaseCommandHandler.cs [hunk 1]
```diff
@@ -27,4 +27,13 @@ protected async Task<OperationResult> Perform(Func<SiteEntity, Task> action, Sit
         await SiteRepository.Save(site, userAccount, cancellationToken);
         return OperationResult.Success();
     }
+
+    protected async Task<OperationResult<SiteId>> PerformWithResultReturn(Func<SiteEntity, Task> action, SiteId siteId, CancellationToken cancellationToken)
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.Domain/Site/CommandHandlers/SiteBaseCommandHandler.cs:31]: Do you need this logic in base command handler as it is used only in one command? Maybe better to remove it here and implement directly in ProvideNameCommandHandler (without inheritance)

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml [hunk 2]
```diff
@@ -23,7 +23,7 @@
         <gds-p>We will also ask you to tell us if you’re using Modern Methods of Construction (MMC). If your site includes a mix of MMC and traditional build homes, you can indicate that here. We will ask for more details when you tell us about the homes on your site.</gds-p>
     </div>
 
-    <a href="@Url.Action("Name", "Site")" id="continue-button">
+    <a href="@Url.Action("Name", "Site")" id="continue-button" data-testid="site-start-continue">
```
Comment (Carq) [inline src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:26]: id is not need anymore and `data-testId` could be "continue-button"

Comment (Antek012) [reply src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml:26]: This is wrong component. LinkButton should be used

**Review by Antek012 — APPROVED** [review]

File: src/HE.Investments.IntegrationTestsFramework/IntegrationTestClient.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ public class IntegrationTestClient
     public IntegrationTestClient(HttpClient client, ILoginData loginData)
     {
         _client = client;
-        _client.BaseAddress = new Uri("https://localhost/");
+        _client.BaseAddress = new Uri("https://localhost:7090/");
```
[the remark this answers is not in the document]
Comment (rafalgajewski) [reply src/HE.Investments.IntegrationTestsFramework/IntegrationTestClient.cs:19]: reverted

File: src/HE.Investment.AHP.Domain/Site/CommandHandlers/SiteBaseCommandHandler.cs (hunk 1 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Domain/Site/CommandHandlers/SiteBaseCommandHandler.cs:31]: moved method to handler as it maka a little benefit in base class indeed

File: src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml (hunk 2 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Views/Site/Start.cshtml]: solved

**Review by Carq — APPROVED** [review]

**Pull request merged**